src/Chirp.CLI.Client/Program.cs

```diff
@@ -64,11 +63,15 @@ static void Main(string[] args)
         }
     }
 
-    static string timeStampConversion(long unix) {
+    public string TimeStampConversion(long unix) {
```

Comment:
Consider if static does actually make sense here. It is an independent method, which likely make sense to have as a function.
Furthermore, it might make sense to have a class with utility functions such as this in the future or even now. It is something to consider:)